src/main/java/hr/inovatrend/petManager/Entities/AnimalType.java

```diff
@@ -0,0 +1,13 @@
+package hr.inovatrend.petManager.Entities;
+
+import lombok.Getter;
+
+@Getter
+public enum AnimalType {
+    CAT,
+    DOG,
+    RODENT,
+    FISH,
+    BIRD,
+    REPTILE
+}
```

Comment:
U enum dodati name kako bi mogla ispisati normalno unutar padajućeg izbornika (selecta).
Primjer:
CAT("Cat"),
DOG("Dog"),.....

private final Stirng name;

AnimalType(String name) {
this.name = name;
}